bin/deepstate/reducer.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import subprocess
+
```

Comment:
Lets make this use `argparse`, and have a copyright notice. Ideally also two-space indent to be consistent with the rest of the python code.